apps/rehabstod/src/components/Table/DiagnosisCell.tsx

```diff
@@ -12,7 +12,7 @@ export function DiagnosisCell({ diagnos, biDiagnoser }: { diagnos: SickLeaveDiag
           <span>
             {diagnos.kod} {diagnos.beskrivning}
           </span>
-          {biDiagnoser.length > 0 && ` ${biDiagnoser.map(({ kod }) => kod).join(', ')}`}
+          {biDiagnoser.length > 0 && `, ${biDiagnoser.map(({ kod }) => kod).join(', ')}`}
```

Comment:
Looking at the sketches, there should be no comma between the main diagnosis code and the sub diagnosis codes

Reply:
Good catch, I'll ask about this in figma.

Reply:
Feedback from UX is that the comma should be there.